### PR TITLE
provider: more envs

### PR DESCRIPTION
**Pull request by greut**
:8ball: or :crystal_ball: ?

### CODE REVIEW

File: website/docs/index.html.markdown [hunk 1]
```diff
@@ -59,15 +59,15 @@ You can specify the following keys using those environment variables.
 
 - `key` - `EXOSCALE_KEY`, `EXOSCALE_API_KEY`, `CLOUDSTACK_KEY`, or `CLOUDSTACK_API_KEY`;
 
-- `secret` - `EXOSCALE_SECRET`, `EXOSCALE_SECRET_KEY`, `CLOUDSTACK_SECRET`, or `CLOUDSTACK_SECRET_KEY`;
+- `secret` - `EXOSCALE_SECRET`, `EXOSCALE_API_SECRET`, `EXOSCALE_SECRET_KEY`, `CLOUDSTACK_SECRET`, or `CLOUDSTACK_SECRET_KEY`;
```
Comment (marcaurele) [inline website/docs/index.html.markdown]: We need to define the one standing out to push people to use the one we really want, something like: - `secret`: `EXOSCALE_SECRET` (but can also read ....).

Then we also need to ensure we read the most desired one first.

Comment (marcaurele) [reply website/docs/index.html.markdown]: My preference will go for `EXOSCALE_ENDPOINT`, `EXOSCALE_KEY`, `EXOSCALE_SECRET`

Comment (greut) [reply website/docs/index.html.markdown]: this documentation, let's undocument some of them to unclutter this page.

File: website/docs/index.html.markdown [hunk 2]
```diff
@@ -57,22 +57,20 @@ token = "..."
...
 
 - `timeout` - `EXOSCALE_TIMEOUT` global timeout;
 
-- `compute_endpoint` - `EXOSCALE_COMPUTE_ENDPOINT`, or `CLOUDSTACK_ENDPOINT`;
+- `compute_endpoint` - `EXOSCALE_ENDPOINT`, or `EXOSCALE_COMPUTE_ENDPOINT`;
 
 - `dns_endpoint` - `EXOSCALE_DNS_ENDPOINT`.
 
-- `gzip_user_data` - `EXOSCALE_GZIP_USER_DATA`.
-
```
Comment (greut) [inline website/docs/index.html.markdown]: @marcaurele less iz moar

**Pull request merged**